src/import/FSHImporter.ts

```diff
@@ -504,6 +586,48 @@ export class FSHImporter extends FSHVisitor {
     return this.aliasAwareValue(ctx.SEQUENCE().getText());
   }
 
+  visitExpression(ctx: pc.ExpressionContext): string {
+    return this.extractString(ctx.STRING());
+  }
+
+  visitXpath(ctx: pc.XpathContext): string {
+    return this.extractString(ctx.STRING());
+  }
+
+  visitSeverity(ctx: pc.SeverityContext): FshCode {
```

Comment:
Could `visitSeverity` use `visitCode` in some way? It seems like we could use `visitCode` to get the `FshCode` and then do the extra checking on `system` and the `code` that is required for severity. There definitely could be something I'm not seeing though that prevents this.

Reply:
Yes, I even made a note to myself that it would be appropriate to refactor those methods, since they share most of their functionality.